### PR TITLE
Added github label rules from other repo

### PR DESCRIPTION
**Pull request by Thraka**
Label rules for dotnet/docs#9128

### CODE REVIEW

File: .ghal.rules.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "labels": {
+    "defaults": [ ":watch: Not Triaged" ],
+    "product": {
+      "(?i).*": "Source - Docs.ms",
+      "(?i)dotnet-csharp$": ":books: Area - C# Guide",
```
Comment (mairaw) [inline .ghal.rules.json]: I think only the first one should be kept (Source - Docs.ms) - all the others don't make sense here

File: .ghal.rules.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      "(?i).*master\/docs\/framework\/wcf.*": ":card_file_box: Technology - WCF",
+      "(?i).*master\/docs\/framework\/winforms.*": ":card_file_box: Technology - WinForms",
+      "(?i).*master\/docs\/standard\/containerized-lifecycle-architecture.*": ":card_file_box: Technology - .NET Architecture,:book: e-book - Docker lifecycle",
+      "(?i).*master\/docs\/standard\/microservices-architecture.*": ":card_file_box: Technology - .NET Architecture,:book: e-book - .NET Microservices",
+      "(?i).*master\/docs\/standard\/modern-web-apps-azure-architecture.*": ":card_file_box: Technology - .NET Architecture,:book: e-book - ASP.NET Core web apps",
+      "(?i).*master\/docs\/standard\/modernize-with-azure-and-containers.*": ":card_file_box: Technology - .NET Architecture,:book: e-book - Modernizing w/ Windows containers",
+      "(?i).*master\/docs\/standard\/security.*": ":card_file_box: Technology - Security",
+      "(?i).*master\/docs\/standard\/serverless-architecture.*": ":card_file_box: Technology - .NET Architecture,:book: e-book - Serverless apps",
+      "(?i).*dotnet-api-docs\/blob\/master\/xml\/system\\.net.*": ":card_file_box: Technology - NCL",
```
Comment (mairaw) [inline .ghal.rules.json]: keep just these for dotnet-api-docs, remove the rest

Comment (Thraka) [reply .ghal.rules.json]: @mairaw Done

**Comment by Thraka** [comment]
@mairaw can i just merge? or do i need to wait for build (which doesn't really apply here)

**Comment by mairaw** [comment]
We tend to wait even when it doesn't apply. Up to you!

**Pull request merged**